### ISSUE TITLE
QOP Immortal cannot be gifted after unbundled

### ISSUE THREAD

**Issue by alvinkim080** [ValveSoftware/Dota2-Gameplay#6958]
### Item name

Bloodfeather Frock

### Description

I got my QOP Ultra Rare immortal pack unbundled and the Shoulder item which is the Bloodfeather Frock can't be gifted but the other 2 items which are the legs and taunt are giftable. I hope to got a fix for this
![QOP1](https://user-images.githubusercontent.com/45165365/218961728-e7c91af8-6495-463c-95e1-1671a0b5b4cd.png)
![QOP2](https://user-images.githubusercontent.com/45165365/218961736-3ace5805-b10f-42f7-89e2-bd213af41dd5.png)
![QOP3](https://user-images.githubusercontent.com/45165365/218961745-78ce335d-f966-49d1-9313-638314eec39c.png)

### Example Match ID and Timestamp

7011482376 - 0:01

**Comment by Aluerie** [comment]
You might want to post this as a comment in #6125 as well, the conversation spiraled into fixing this exact issue. Preferably with item ID or, at least, account ID. 

---

and btw, interesting, mine does have the option x)

![image](https://user-images.githubusercontent.com/33165440/218974050-79008228-50a6-4de9-b279-5faf03da97d3.png)

**Comment by Aluerie** [comment]
i mean you still did not include neither item ID nor account ID. 

how valve dev gonna check your exact item.

**Comment by alvinkim080** [comment]
Edited, thanks for the feedback

**Comment by alvinkim080** [comment]
> You might want to post this as a comment in #6125 as well, the conversation spiraled into fixing this exact issue. Preferably with item ID or, at least, account ID.
> 
> and btw, interesting, mine does have the option x)
> 
> ![image](https://user-images.githubusercontent.com/33165440/218974050-79008228-50a6-4de9-b279-5faf03da97d3.png)

i'm not sure why but maybe because I changed the name of the item few days ago

**Comment by alvinkim080** [comment]
> Did you try resetting the name?

Yup I did reset it when the legs and taunt became giftable but still not fixed
